src/Enzo/enzo_EnzoMethodFeedbackSTARSS.cpp

```diff
@@ -1053,11 +1025,11 @@ void EnzoMethodFeedbackSTARSS::deposit_feedback (Block * block,
   double pTerminal;
   if (Z_Zsun > 0.01) {
     // Thornton, 1998, M_s * V_s, eqs 22, 23, 33, 34
-    pTerminal = 1.6272e5 * pow(ejectaEnergy/1e51, 13./14.) * pow(n_mean, -0.25) * pow(Z_Zsun, -0.36);
+    pTerminal = 2.7298e5 * pow(ejectaEnergy/1e51, 13./14.) * pow(n_mean, -0.12) * pow(Z_Zsun, -0.14); // Msun km/s
```

Comment:
Where did the changes in normalization and slope come from in `p_free` and `pTerminal`?  Are they different cases in Thorton (1998)? Also I couldn't find this paper / book.